profiling/src/php_ffi.c

```diff
@@ -233,3 +233,48 @@ void ddog_php_test_free_fake_zend_execute_data(zend_execute_data *execute_data)
     free(execute_data);
 }
 #endif
+
+uint8_t *opcache_handle = NULL;
```

Comment:
Is there a reason why this is uint8_t? The proper value for dynamic library handles is `void *`. This only works because C allows automatic conversion between void * and any other pointer type...